Assets/Scripts/Augment/BulletModifiers/KnockbackOnShotModifier.cs

```diff
@@ -17,7 +17,6 @@ private void Awake()
         gunController = transform.parent.GetComponent<GunController>();
         if (!gunController)
```

Comment:
I would prefer we rewrote all these guncontroller checks to the abbreviated no-body version, like:
`
if (!gunController)
   return;
`
But I consider this a nitpick you don't need to do here unless you want to, as ideally later we won't need this pattern if we resolve the now infamous #126